packages/block-editor/src/components/rich-text/style.scss

```diff
@@ -15,7 +15,7 @@
 	// breaking spaces in between words. If also prevent Firefox from inserting
 	// a trailing `br` node to visualise any trailing space, causing the element
 	// to be saved.
-	white-space: pre-wrap;
+	white-space: pre-wrap !important;
```

Comment:
Safe to say this is `!important` because it breaks spectacularly when any other style takes precedence?

Reply:
Yes, I would say that this is the only CSS rule required for RichText to work properly. It might even be better to insert it inline? Not sure.